WMF Framework/LocationManagerProtocol.swift

```diff
@@ -0,0 +1,34 @@
+import CoreLocation
```

Comment:
Move after "Add logs to LocationManager"

Reply:
maybe merge with "Add ObjC LocationManagerFactory" and rename to "Add ObjC support for LocationManager"